src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.CSharp.Analyzers/UrlHelperAnalyzer.cs

```diff
@@ -86,10 +89,29 @@ private static string GetReplacementType(SyntaxNode node)
             return parent switch
             {
                 QualifiedNameSyntax => GetReplacementType(parent),
-                ObjectCreationExpressionSyntax => AspNetCoreUrlHelperName, // Object creation requires HtmlHelper
-                MemberAccessExpressionSyntax => AspNetCoreUrlHelperName, // Member access requires HtmlHelper
-                _ => AspNetCoreIUrlHelperName // Default to IHtmlHelper
+                ObjectCreationExpressionSyntax => AspNetCoreUrlHelperName, // Object creation requires UrlHelper
+                MemberAccessExpressionSyntax => AspNetCoreUrlHelperName, // Member access requires UrlHelper
+                _ => AspNetCoreIUrlHelperName // Default to IUrlHelper
             };
         }
+
+        private static SyntaxNode GetQualifiedName(SyntaxNode name)
```

Comment:
I think we just added a helper method for this. see SyntaxNodeExtensions.cs

Reply:
Yep. Good catch. Fixed.